ci/test.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash -ex
 
-cd "${PWD}/tests"
+cd "${PWD}/tests/${1}"
```

Comment:
It would be nice to assign this to a variable and echo it out so that we know what's being run. Also, what happens when nothing is provided?

Reply:
Good catch - this should have a default behavior in addition to echoing what it's going to run. I'll add that!